vowpalwabbit/array_parameters_dense.h

```diff
@@ -75,7 +75,7 @@ class dense_parameters
   const_iterator cbegin() { return const_iterator(_begin, _begin, stride()); }
   const_iterator cend() { return const_iterator(_begin + _weight_mask + 1, _begin, stride()); }
 
-  inline weight& operator[](size_t i) const { return _begin[i & _weight_mask]; }
+  inline weight& operator[](uint64_t i) const { return _begin[i & _weight_mask]; }
```

Comment:
size_t - please check earlier comment.

Reply:
see comment above

Reply:
see comment above

---
In reply to: [188074324](https://github.com/JohnLangford/vowpal_wabbit/pull/1481#discussion_r188074324) [](ancestors = 188074324)